File: analyst/docs/competitors/c++/SquareRoot.cpp
#include <iostream> 
#include <cmath> 
#include <iomanip> 
int main() { 
	double inputNumber;
	std::cout << "Enter a real number: "; 
	std::cin >> inputNumber;
	if (inputNumber < 0.0)
	{
		std::cout << "ERROR" << std::endl; 
	} 
	else 
	{ 
		double root = std::sqrt(inputNumber);
		std::cout << "Square root: " << std::fixed << std::setprecision(2) << root << std::endl; 
	} 
    return 0; 
}

File: analyst/docs/competitors/pascal/SquareRoot/SquareRoot.pas
program SquareRoot;
var
  inputNumber, root: real;
begin
  writeln('Enter a real number: ');
  read(inputNumber);
  if inputNumber < 0
  then
    begin
      writeln('ERROR');
    end
  else
    begin
      root := sqrt(inputNumber);
      writeln('Square root: ', root:0:2);
    end;
  readln;
end.
